src/exportJarSteps/GenerateJarExecutor.ts

```diff
@@ -142,6 +164,9 @@ export class GenerateJarExecutor implements IExportJarStepExecutor {
                         if (_.isEmpty(pickBox.selectedItems)) {
                             return;
                         }
+                        if (!stepMetadata.classpaths) {
```

Comment:
Does `undefined` and `[]` have different meaning here?

Reply:
Yes. The following steps will add some strings into the `stepMetadata.classpaths`, so it can be `[]`(in fact, it should be `[]`). But it's illegal to insert an element into an `undefined`.